src/content/posts/learning-astro.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: ../../layouts/PostLayout.astro # Always use this layout, it's so the post gets properly styled
+title: The title of the post
+excerpt: A short description of the post
+author: your-author-username
+createdAt: Month Day, Year
```

Comment:
Please add the date here.

Reply:
Hey check the commit now sorry i overlooked that part :)